aries-site/src/pages/templates/lists.mdx

```diff
@@ -1,16 +1,109 @@
 import { Example } from '../../layouts';
-import { ListScreenExample } from '../../examples';
+import {
+  ListIconIdentifierExample,
+  ListImageIdentifierExample,
+  ListNameOptionActionExample,
+  ListNameActionExample,
+  ListNameDescriptionOptionExample,
+  ListScreenExample,
+} from '../../examples';
 
 HPE Design System list view templates are go-to patterns for displaying homogeneous data such as services, devices, users, and more.
-
-List views are optimized for scanability and reading comprehension. Each list item provides users focussed information and identity labels to aid selection, decision making, and action.
-
-## Manage Users Screen
+List views are optimized for scanability and reading comprehension. Each list item provides users focused information and identity labels to aid selection, decision making, and action.
 
 <Example
   code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/list-views/ListScreenExample.js"
+  docs="https://v2.grommet.io/list?theme=hpe#props"
   template
   width="100%"
 >
   <ListScreenExample />
 </Example>
+
+## Guidance
+
+Lists are an effective way to display dense content to users. Here are some recommendations on how to make the data easily consumable.
+
+**Best Practices:**
+
+- Be consistent with layout and visual treatments for all list items.
+- Keep list item descriptions brief and concise.
+- Use Icons or Images in list items to help draw attention and support scanning.
+- Maintain sufficient space between list items or consider using dividers between rows.
+- Sort lists in a logical order such as alphabetically, numerically, chronologically, or by user preference.
+
+**Lists Types:**
+
+- Single-line lists are optimal for scan-ability, especially with the addition of an image.
+- Two-line lists provide a primary and secondary text which can be helpful when more detail is needed.
+- Three-line lists of a primary text and two lines of secondary text are not recommended.
+- Grid lists are made of image tiles and require more space.
+
+## Variants
+
+Below are some common list variations that demonstrate how to combine visual content, text, and list actions within the component.
+
+### Image + Name + Description + Action
+
+Two-line list with an image, primary title, description, and an action button.
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/list-views/ListImageIdentifierExample.js"
+  docs="https://v2.grommet.io/list?theme=hpe#props"
+  figma="https://www.figma.com/file/tMYTr17r6hhGajSlNRiPje/HPE-List-Component?node-id=19%3A135"
+  width="100%"
+>
+  <ListImageIdentifierExample />
```

Comment:
Fullscreen mode width seems to be too narrow:
![Screen Shot 2020-07-06 at 1 30 58 PM](https://user-images.githubusercontent.com/1756948/86632858-27f6c880-bf8d-11ea-9fa7-d72de1529086.png)